nntrainer/tensor/tensor.h

```diff
@@ -469,6 +469,12 @@ class Tensor {
   Tensor &dot(Tensor const &m, Tensor &output, bool trans = false,
               bool trans_m = false, float beta = 0.0f) const;
 
+  /**
+   * @brief transpose batch and channel and
+   * flip the given Tensor horizontal and vertical
+   */
+  Tensor convFlip() const;
```

Comment:
how about rename convFlip to convTransFlip?

Reply:
Done

Reply:
This function is deleted, the patch has been done to avoid this memcopy